lib/resources/r4/related_person_patch.yaml

```diff
@@ -30,3 +30,36 @@ operations:
         <li>The <code>value.coding</code> cannot contain a standard system.</li>
         <li>Duplicate relationships may be added, but will only be represented once on the Resource.</li>
       </ul>
+
+  - name: add-address
+    path: /address/-
+    operation: add
+    type: Address
+    url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson.address
+    description: Add a new address for the related person.
+    example: |
+      {
+        "op": "add",
+        "path": "/address/-",
+        "value": {
+          "use": "home",
+          "line": [
+            "1212 Faircastle Drive",
+            "Apartment 406"
+          ],
+          "city": "KC",
+          "district": "Jackson",
+          "state": "KS",
+          "postalCode": "64199",
+          "country": "United States of America",
+          "period": {
+            "start": "2012-05-17T15:33:18.000Z"
+          }
+        }
+      }
+    note: >
+      <ul>
+        <li>Must not contain the `text` field.</li>
+        <li>Must contain the `use` field, and must include a time component with a timezone when specifying a `period`.</li>
```

Comment:
Let's separate the use comment and the period comment to different elements in the list.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/536/commits/85d1061bd3edc9b5d4b6f6bf7c73058418c04bac